### ISSUE TITLE
add tab after clicking play

### ISSUE THREAD

**Issue by usefree** [fossfreedom/radio-browser#14]
Please add possibility to specify stream link manually. 
If the link is broken, then nothing happend after pressing control buttons.
##

**Comment by fossfreedom** [comment]
talk me through how this works - any screenshots/mockups of what you are asking for?  I dont really understand what the enhancement you are asking for.

Can you give me the URL's of where a "link is broken, then nothing happend after pressing control buttons"

**Comment by usefree** [comment]
I search "di.fm", in search results find "lounge" and press "play" - nothing become playing. That is because of wrong Stream URL and i am asking for ability to customize this URL. 
Screenshot attached.
![radiostation-browser-play-record](https://cloud.githubusercontent.com/assets/1050026/10782998/2d436a90-7d5d-11e5-8e05-609dd40d42fb.png)

**Comment by fossfreedom** [comment]
Click on "Board (16)" above and the info box should change and display a button which allows you to submit a new url .

I cant remember if it automatically appears or if you click the redownload feed button to pull down the latest radio stations including the one you submitted.

**Comment by usefree** [comment]
if i click on "Board" there is nothing appeares in Infobox section:
![radiostation-browser-play-record-1](https://cloud.githubusercontent.com/assets/1050026/10790187/8a000220-7d8a-11e5-9c88-a01f50fdee5d.png)

**Comment by fossfreedom** [comment]
@usefree 

  well found.  Looks like this wasnt implemented correctly ... it was definitely not converted correctly from the original code base.

I've tidied up the code - you should now be able to submit a new url when you click "board"

**Comment by usefree** [comment]
@fossfreedom , i cloned repo, paste files to ..RB/Plugins.., restart RB, but there are no changes in GUI. The same situation as before. 
And what should be after clicking on button "Post new station" on "Radiostation List" Tab? Nothing happened after pressing it.
![radiostation-browser-play-record-2](https://cloud.githubusercontent.com/assets/1050026/10813485/22ebc49a-7e28-11e5-8da5-4022aa6ff965.png)

**Comment by fossfreedom** [comment]
not sure by what you mean "paste files" ...

please install as per the README

git clone https://github.com/fossfreedom/radio-browser -b rb3
cd radio-browser
./install.sh

If there is still an issue, please launch rhythmbox via

rhythmbox -d

click the post new station button and let me know what the output of the last 100 lines produced in the terminal - use paste.ubuntu.com and paste a link here.

**Comment by usefree** [comment]
running ./install.sh not working/
syntax errors at lines:
<<<<<<< HEAD
=======
>>>>>>> 79c4ac69c682a86c63e3e14c48cf7c9431336c68

**Comment by usefree** [comment]
@fossfreedom  yeah!!! It is working now! I succesfully installed plugin, added radiostation and initiated recording. Then found it on Board. Thanks. It would be wonderfull if You make an adding a tab to GUI after pressing button Play, not only after pressing Record.
![radiostation-browser-play-record-3](https://cloud.githubusercontent.com/assets/1050026/10817441/75f9d28e-7e42-11e5-9a7b-ef697a74257a.png)

**Comment by fossfreedom** [comment]
ok,

  excellent news.  I'll update the PPA later

If you want to revert to the PPA version just run

./install.sh --uninstall

I'll add the request to add a station tab as a wish list item that either myself or someone else can implement at a later date.

If you have any programming experience and would like to make any changes - big or small - feel free to submit a pull request.
